### ISSUE TITLE
SA1121 UseBuiltInTypeAlias incorrectly reported in nameof

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#387]
The following statement incorrectly results in SA1121.

```csharp
string objectName = nameof(System.Object);
```

**Comment by sharwell** [comment]
@pdelvo I assigned this to you since you implemented the initial diagnostic.
